test/web-platform-tests/to-upstream/dom/events/Event-dispatch-throwing.html

```diff
@@ -0,0 +1,30 @@
+﻿<!DOCTYPE html>
+<meta charset="UTF-8">
+<title>Throwing in event listeners</title>
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+<div id="log"></div>
+<script>
+  "use strict";
+
+  setup({ allow_uncaught_exception: true });
+
+  test(() => {
+    let errorEvents = 0;
+    document.defaultView.addEventListener("error", event => {
```

Comment:
In addition to moving this file into the correct location, it's best to update this to just `window.addEventListener`